cli/azd/cmd/cobra_builder.go

```diff
@@ -189,6 +189,9 @@ func (cb *CobraBuilder) configureActionResolver(cmd *cobra.Command, descriptor *
 						output.WithHighLightFormat(suggestionErr.Suggestion))
 				}
 			}
+
+			// Stop the spinner always to un-hide cursor
+			console.StopSpinner(ctx, "", input.Step)
```

Comment:
is this fixing: https://github.com/Azure/azure-dev/issues/2194 ??

Reply:
I believe so, forgot to tag it.

Reply:
Updated the PR description